### PR TITLE
optimize README

### PR DESCRIPTION
**Pull request by Presburger**

### CODE REVIEW

**Comment by Presburger** [comment]
kind/improvement

File: README.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ $ export PATH=$PATH:$HOME/.local/bin
 
 ```bash
 $ mkdir build && cd build
+#add conan remote
+$ conan remote add default-conan-local https://milvus01.jfrog.io/artifactory/api/conan/default-conan-local
```
Comment (liliu-z) [inline README.md:43]: I saw this in our install script

**Pull request merged**